docs/source/api/session.rst

```diff
@@ -0,0 +1,16 @@
+`Session`
+==========
+
+Session is an extension of `tf.Session` that lets the graph run in a secure manner.
```

Comment:
Is it an extension or a wrapper around tfe.Session? We could just document that it wraps tf.Session and handles configuring the session for secure computation with the chosen protocol.